addons/mail/static/src/js/views/activity/activity_renderer.js

```diff
@@ -78,30 +107,60 @@ var ActivityRenderer = AbstractRenderer.extend({
      */
     _renderRow: function (data) {
         var self = this;
-        var res_id = data[0];
-        var name = data[1];
-        var $nameTD = $('<td>')
-            .addClass("o_res_name_cell")
-            .html(name)
-            .data('res-id', res_id);
+        var res_id = data.id;
+        if (data.currency_id) {
+            data.subtitle_currency = field_utils.format.monetary(data.subtitle, {}, {currency_id: data.currency_id});
+        }
+        var $nameTR = $(QWeb.render('mail.ActivityViewRowSidebar', {data: data}));
         var $cells = _.map(this.state.data.activity_types, function (node) {
-            var $td = $('<td>').addClass("o_activity_summary_cell");
             var activity_type_id = node[0];
             var activity_group = self.state.data.grouped_activities[res_id][activity_type_id];
             activity_group = activity_group || {count: 0, ids: [], state: false};
+
+            var $td = $(QWeb.render('mail.ActivityViewRow', {
+                data: data,
+                activity_group: activity_group,
+                activity_type_id: activity_type_id,
+            }));
             if (activity_group.state) {
-                $td.addClass(activity_group.state);
+                var record = self._getkanbanActivityData(activity_group, res_id);
+                var KanbanActivity = field_registry.get('kanban_activity');
+                var widget = new KanbanActivity(self, "activity_ids", record, {});
+                widget.appendTo($td);
+                // replace clock by closest deadline
+                var $date = $('<div class="o_closest_deadline">');
+                var date = new Date(activity_group.o_closest_deadline);
+                // To remove year only if current year
+                if (moment().year() === moment(date).year()) {
+                    $date.text(date.toLocaleDateString(moment().localeData()._abbr, {day: 'numeric', month: 'short'}));
```

Comment:
Rely on native locale detection (`window.navigator.language`) instead of momentjs' way of doing it.
By the way, if you use momentjs, you should use `moment.locale()`